### PR TITLE
add unbiased mode to the MSAA render pass.

### PR DESCRIPTION
**Pull request by bhouston**
One of the issues with the manual MSAA render pass I've noticed is that it uses the same sample weight for each render.  When rendering to a 8-bit-per channel render buffer this means that each sample will likely be rounded the same way, thus accumulating round off errors at high sample counts.

Thus I have added an "unbiased' mode to the ManualMSAARenderPass that varies the sample weight in a uniform distribution such that the rounding errors cancel each other out instead of accumulating.  There is no speed cost and no need to get render into a higher bit-depth render target.

I've created an example that allows one to turn on/off the unbiased MSAA mode.

Here is the old 32 sample MSAA result that accumulated round-off errors:

image

Here is what the new unbiased MSAA produces with 32 samples:

image

### CODE REVIEW

File: examples/js/postprocessing/ManualMSAARenderPass.js [hunk 1]
```diff
@@ -77,27 +78,39 @@ Object.assign( THREE.ManualMSAARenderPass.prototype, {
...
+			this.copyUniforms[ "opacity" ].value = sampleWeight;
+
 			renderer.render( this.scene, this.camera, this.sampleRenderTarget, true );
 			renderer.render( this.scene2, this.camera2, writeBuffer, (i === 0) );
 
 		}
 
-		// reset jitter to nothing.	TODO: add support for orthogonal cameras
-		if ( camera.setViewOffset ) camera.setViewOffset( undefined, undefined, undefined, undefined, undefined, undefined );
+		if ( this.camera.setViewOffset ) this.camera.setViewOffset( undefined, undefined, undefined, undefined, undefined, undefined );
```
Comment (mrdoob) [inline examples/js/postprocessing/ManualMSAARenderPass.js]: Same as `this.camera.setViewOffset()`. We could also do `this.camera.clearViewOffset()` for clarity.

Comment (bhouston) [inline examples/js/postprocessing/ManualMSAARenderPass.js]: I've added this.camera.clearViewOffset() per your recommendations.  It is clearer.

**Comment by mrdoob** [comment]
Thanks!

**Comment by bhouston** [comment]
Thank you for being open to the improvements. :)

**Pull request merged**